sysdeps/managarm/generic/file.cpp

```diff
@@ -4366,5 +4366,10 @@ int sys_gethostname(char *buffer, size_t bufsize) {
 	return 0;
 }
 
+int sys_fsync(int) {
+	mlibc::infoLogger() << "mlibc: fsync is a stub" << frg::endlog;
+	return 0;
```

Comment:
This can also be handled by a sysdep function.

Reply:
This is a sysdep tho. All other places agreed, unsure about `setgroups`.

Reply:
Oh, alright. I misread that.